x-pack/plugins/enterprise_search/public/applications/app_search/components/library/library.tsx

```diff
@@ -50,6 +57,17 @@ export const Library: React.FC = () => {
     },
   };
 
+  const schema = {
+    title: 'text' as SchemaTypes,
+    description: 'text' as SchemaTypes,
+    date_established: 'date' as SchemaTypes,
+    location: 'geolocation' as SchemaTypes,
+    states: 'text' as SchemaTypes,
+    visitors: 'number' as SchemaTypes,
+    size: 'number' as SchemaTypes,
+    length: 'number' as SchemaTypes,
+  };
```

Comment:
Minor suggestion if you don't want to repeat `as SchemaTypes` over and over again
```suggestion
  const schema = {
    title: 'text',
    description: 'text',
    date_established: 'date',
    location: 'geolocation',
    states: 'text',
    visitors: 'number',
    size: 'number',
    length: 'number',
  } as Schema;
```

Reply:
3ea49844d515ec9055519731a5c55c71f441b0c3

Reply:
3ea49844d515ec9055519731a5c55c71f441b0c3